src/sidebar/util/test/thread-annotations-test.js

```diff
@@ -1,6 +1,7 @@
 import * as annotationFixtures from '../../test/annotation-fixtures';
 import uiConstants from '../../ui-constants';
 import threadAnnotations from '../thread-annotations';
+import { sortFns } from '../thread-sorting';
```

Comment:
The `Fns` abbreviation here is a bit of a hangover from where Angular used to use it quite widely. Perhaps `sorters` (object name) / `thread-sorters` (module name) would be more in line with the naming conventions used for other modules.

Reply:
I've changed it.